examples/2d/many_sprites.rs

```diff
@@ -0,0 +1,90 @@
+use bevy::{
+    diagnostic::{FrameTimeDiagnosticsPlugin, LogDiagnosticsPlugin},
+    math::Quat,
+    prelude::*,
+};
+
+const CAMERA_SPEED: f32 = 1000.0;
+
+pub struct PrintTimer(Timer);
+pub struct Position(Transform);
+
+///This example is for performance testing purposes.
+///See https://github.com/bevyengine/bevy/pull/1492
+fn main() {
+    App::build()
+        .add_plugin(LogDiagnosticsPlugin::default())
+        .add_plugin(FrameTimeDiagnosticsPlugin::default())
+        .insert_resource(bevy::log::LogSettings {
+            level: bevy::log::Level::DEBUG,
+            ..Default::default()
+        })
```

Comment:
Why is this required? By default the `INFO` of the `LogDiagnosticPlugin` is displayed properly

```suggestion
```